Samples/Objective-C/nhubsample-refresh/nhubsample-refresh.xcodeproj/project.pbxproj

```diff
@@ -353,12 +353,14 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = S4V3D7CHJR;
 				INFOPLIST_FILE = "nhubsample-refresh/Info.plist";
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.example.nhubsample-refresh";
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
`Scripts/VerifyNoBuildSettings.swift` script requires that all settings like this are placed in .xcconfig files instead of directly in project. Apply this script and move all settings or just remove it if you do not need it